gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -278,18 +314,28 @@ public void copy(Path source, Path target, CopyOption... options) throws IOExcep
       } else if (option instanceof CloudStorageOption) {
         if (option instanceof OptionBlockSize) {
           blockSize = ((OptionBlockSize) option).size();
+        } else if (option instanceof AutoValue_OptionMimeType) {
+          tgtInfoBuilder.contentType(((AutoValue_OptionMimeType) option).mimeType());
+          setContentType = true;
+        } else if (option instanceof AutoValue_OptionCacheControl) {
+          tgtInfoBuilder.cacheControl(((AutoValue_OptionCacheControl) option).cacheControl());
+          setCacheControl = true;
+        } else if (option instanceof AutoValue_OptionBlockSize) {
+          blockSize = ((AutoValue_OptionBlockSize) option).size();
+        } else {
```

Comment:
I'm confused. There is no call to either buildFileOptions or copyFileOptions in the snippet you're showing. In fact I can't find those anywhere in the code. Would you mind please clarifying?